docs/references/subcommands/analyze.md

```diff
@@ -44,6 +58,15 @@ fossa analyze --json
 {"project":{"name":"custom@new-project","branch":"master","revision":"123","url":"https://app.fossa.com/projects/custom+<org-id>/new-project/refs/branch/master/123","id":"custom+<org-id>/new-project$123"}}
 ```
 
+### Vendored Dependencies
+
+The Vendored Dependencies feature allows you to scan for licenses directly in your code. For more information, please see the [Vendored Dependencies documentation](../../features/vendored-dependencies.md).
+
+| Name                                      | Description                                                                                                                                                                           |
+|-------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--force-vendored-dependency-scan-method` | Force the vendored dependency scan method. The options are 'CLILicenseScan' or 'ArchiveUpload'. 'CLILicenseScan' is usually the default unless your organization has overridden this. |
```

Comment:
Is there a place a user can find out what their organization has set? If so and it's simple to point it out, it might be good to mention it but consider this optional.

Reply:
There's not a great way to do this. 

I wonder if we need a "show org config" command to do this. We're building up a reasonably large set of configs, and it'd be nice to show this to the user somehow. (and by "we", I mean "mostly, maybe 100%, added by me")

We should also add a section to the debug bundle for this.